.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
 sudo: false
-cache: pip
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/virtualenv
```

Comment:
I think I don't understand the difference between caching `.cache/pip` and using a download cache. If we cache both when only one is necessary then that's more stuff to copy on startup, so it's unclear to me what the tradeoff is. What's your take on it? For now, I think https://github.com/dimagi/commcare-hq/pull/8608 is good start?